### PR TITLE
Tagged Installation Instructions

### LINKED ISSUE #830: Update readme instructions for git installation with current tag
**Is your feature request related to a problem? Please describe.**
Even though we have versioning in the `start_docker.sh` script, the instructions for cloning the repo point to the current version of main. This means that the instructions are not backwards compatible, and checking out a previous version of the repo won't result in the previous version of code being installed. 

**Describe the solution you'd like**
Update the readme to have a specific tagged version of the codebase which gets installed, which we'll update (along with tag in start_docker.sh) each time we put out a new release.  We'll also need to update the developer docs with a note to update this section of the readme for each new release

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #830.

**How did you implement your changes**

Added directions to install a specific release of Ark.

**Remaining issues**

N/A.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,14 +83,18 @@ After the Pixie Pipeline, the user can inspect and fine tune their results with
...
+We recommend using the latest release of `ark`. You can find all the versions available in the [Releases Section](https://github.com/angelolab/ark-analysis/releases).
 Open terminal and navigate to where you want the code stored. 
 
-Then input the command:
+Currently, the latest release is `v0.4.4`.
+Then install the latest release with:
 
 ```
-git clone https://github.com/angelolab/ark-analysis.git
+git clone https://github.com/angelolab/ark-analysis.git --depth 1 --branch v0.4.4
```
Comment (ngreenwald) [inline README.md:93]: Can you try it out to make sure this works as expected, i.e. previous docker gets pulled and specified from start_docker.sh script, etc etc

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, will you also update the developer docs for making a new release

File: README.md (hunk 1 above)
Comment (srivarra) [reply README.md:93]: yup yup

Comment (srivarra) [reply README.md:93]: Yeah the `VERSION` in `start_docker_sh` is `v0.4.4`, so it runs as expected.

**Review by ngreenwald — APPROVED** [review]
Looks good!

**Pull request merged**